src/zino/tasks/juniperalarmtask.py

```diff
@@ -40,10 +46,14 @@ async def _get_juniper_alarms(self):
 
         if type(yellow_alarm_count) is not int or type(red_alarm_count) is not int:
             _logger.error(
-                "Device %s returns alarm count not of type int, type yellow alarm count: %s, type red alarm count: %s",
+                "Device %s returns alarm count not of type int. "
+                "Yellow alarm count: type %s, value %s. "
+                "Red alarm count: type %s, value %s.",
```

Comment:
```suggestion
                "Yellow alarm count: type %s, value %r. "
                "Red alarm count: type %s, value %r.",
```

Also, given @stveit's comments, we might consider moving the nitty-gritty details to a separate debug log statement...